docs/OpenCLCOnVulkan.md

```diff
@@ -33,9 +33,17 @@ The SPIR-V as produced from the OpenCL C language can make use of the following
 capabilities:
 
 - `Shader` as we are targeting the OpenCL C language at a Vulkan implementation.
+- `VariablePointersStorageBuffer`, from the _SPRV\_KHR\_variable\_pointers_ extension.
 - `VariablePointers`, from the _SPV\_KHR\_variable\_pointers_ extension.
-  - *Note*: The compiler always emits code that depends on `VariablePointers`
-    even though there might some cases where it is not strictly needed.
+ - *Note*: the compiler attempts to add the minimal variable pointers capability required.
+- `Int16` if short or ushort types are used.
+- `Int64` if long or ulong types are used.
+- `Float16` if the half type is used.
```

Comment:
Need another space before the -
Check out the rendering at https://github.com/alan-baker/clspv/blob/docs-update/docs/OpenCLCOnVulkan.md#spir-v-features